ebos/eclwriter.hh

```diff
@@ -410,7 +413,8 @@ public:
                                      eclOutputModule_->getBlockData(),
                                      eclOutputModule_->getWBPData(),
                                      localWellData,
-                                     localGroupAndNetworkData);
+                                     localGroupAndNetworkData,
+                                     {});
```

Comment:
I do not understand why you do not add the aquifer data here?

Reply:
The aquifer data is only used in evalSummary, it is not used in writeOutput. so there is no need to communicate it here.

Reply:
Thanks, I assume that is because we only output aquifer data in summary-type output?

Reply:
For now, yes. Adding aquifer restart output is on my list of things to do in the new year.